packages/rs-dpp/src/identity/state_transition/asset_lock_proof/instant/methods/validate_structure/v0/mod.rs

```diff
@@ -0,0 +1,35 @@
+use platform_version::version::PlatformVersion;
+use crate::consensus::basic::identity::IdentityAssetLockProofLockedTransactionMismatchError;
+use crate::identity::state_transition::asset_lock_proof::InstantAssetLockProof;
+use crate::identity::state_transition::asset_lock_proof::validate_asset_lock_transaction_structure::validate_asset_lock_transaction_structure;
+use crate::ProtocolError;
+use crate::validation::SimpleConsensusValidationResult;
+
+pub(in crate::identity::state_transition::asset_lock_proof::instant) fn validate_instant_asset_lock_proof_structure_v0(
```

Comment:
pub(super) is better here.

Reply:
super won't work here because structure define two modules above